policyengine_us/tests/policy/baseline/gov/states/nd/tax/income/nd_mpc.yaml

```diff
@@ -51,3 +51,56 @@
     nd_taxable_income: 354_100
     nd_income_tax_before_credits: 7_356.02
     nd_mpc: 208  # capped at maximum credit amount
+
+
+- name: ND marriage-penalty credit unit test 3
```

Comment:
could you make the test names descriptive of the tests that you added? What case are we testing and why is it relevant?

Reply:
all test files

Reply:
Ok, will do it right now.